app/components/agenda-manager/edit.hbs

```diff
@@ -53,20 +53,30 @@
               {{t "manageAgendaZittingModalMove.title"}}
             </Form.SelectLocation>
           {{/if}}
-          {{#unless this.isNew}}
-          {{!-- TODO: don't allow delete on published agendapoints  --}}
-            <AgendaManager::Delete @onDelete={{fn (perform this.deleteTask) @itemToEdit}} @loading={{this.deleteTask.isRunning}}/>
-          {{/unless}}
         </div>
       {{/if}}
     </Modal.Body>
     <Modal.Footer>
+      <AuToolbar>
       <AuButtonGroup>
         <Form.Submit>{{t "manageAgendaZittingModal.saveButton"}}</Form.Submit>
         <AuButton @skin="secondary" {{on "click" this.cancel}}>
           {{t "manageAgendaZittingModal.cancelButton"}}
         </AuButton>
       </AuButtonGroup>
+      {{#unless this.isNew}}
+        {{!-- TODO: don't allow delete on published agendapoints  --}}
```

Comment:
since this is a pretty simple check to add (and deleting published agendapoints wasn't possible before), why not just add it?

Reply:
We can't edit published agenda points so we cannot delete them, I just tested it, I will delete the comment